Frontend/frontend/package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "bootstrap": "^4.4.1",
```

Comment:
Are we using bootstrap or react bootstrap

Reply:
react-bootstrap, but it uses bootstrap's css files